Bottom: 4b810000b83b48e6ea63cba940ba9490dfaba4e9
Top:    c31aa5df3e9296c2dec1f9060759dc46607015d5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-01 00:13:36 +0100

Refresh of hid-gtk-remove-unused-variable

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index a28b19a..64a12a8 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -255,7 +255,6 @@ void
 ghid_use_mask (int use_it)
 {
   static int mask_seq_id = 0;
-  static GdkDrawable *old;
   GdkColor color;
   render_priv *priv = gport->render_priv;
